Datahub.Web/Models/SearchResult.cs

```diff
@@ -37,11 +37,11 @@ public class SearchResult
         [Text(Name = "data_type")]
         public string DataType { get; set; }
 
-        [Nested]
-        [PropertyName("datahub_keywords")]
-        public Keyword[] DatahubKeywords { get; set; }
+        // [Nested]
+        // [PropertyName("datahub_keywords")]
+        // public Keyword[] DatahubKeywords { get; set; }
 
-        [GeoShape(Name = "footprint")]
-        public IGeoShape Footprint { get; set; }
+        // [GeoShape(Name = "footprint")]
+        // public IGeoShape Footprint { get; set; }
```

Comment:
Fair enough to comment this out  if we aren't using it right now, but didn't know this was causing any complications

Reply:
It's not actually used / unsupported so it's confusing. Could add a comment I guess. Commenting-out is WiP